### DISCUSSION TITLE
Microsoft Azure 13-17 years old

### DISCUSSION THREAD

**Discussion by Sandro642** [community/community#49009] [GitHub Education] [answered]
Hi, I would like to benefit from the Microsoft Azure service for free but for that I have to give a 25 digit code, do you know where I can find it.

THANKS

**Comment by samus-aran** [comment, marked as answer]
Hi @Sandro642 👋 

If this [Azure verification page](https://signup.azure.com/studentverification?offerType=1) does not work then we would suggest you reach out to [Azure Support](https://support.microsoft.com/en-us/topic/2315e669-8b1f-493b-5fb1-d88a8736ffe4) if you are having problems redeeming your Azure benefits.

**Reply by DiscoverProgramming** [reply]
Sorry about replying to a year+ old thread, but what did Azure Support say? What was the solution?
